src/org/recompile/mobile/PlatformPlayer.java

```diff
@@ -56,17 +56,17 @@ public PlatformPlayer(InputStream stream, String type)
 		}
 		else
 		{
-			if(type.equals("audio/midi") || type.equals("sp-midi") || type.equals("audio/spmidi"))
+			if(type.equals("audio/mid") || type.equals("Audio/midi") || type.equals("audio/midi") || type.equals("sp-midi") || type.equals("audio/spmidi"))

```

Comment:
Rather than check for random differences in capitalization as we find them, why not normalize the string by forcing it to lower case before checking?